ambry-replication/src/main/java/com/github/ambry/replication/RemoteReplicaInfo.java

```diff
@@ -67,6 +68,9 @@ public class RemoteReplicaInfo {
   private long localLagFromRemoteStore = -1;
   private long reEnableReplicationTime = 0;
   private ReplicaThread replicaThread;
+  private long replicationRetryCount;
+  // Configurable
+  protected final int MAX_REPLICATION_RETRY_COUNT;
```

Comment:
all cap variable names are used for static constant, please change this variable name to something like maxReplicationRetryCount.

Reply:
done